File: 0006_Alphabetical_Addition.py
def add_letters(*args):
    alphabet = 'abcdefghijklmnopqrstuvwxyz'
    number = [alphabet.index(number)+1 for number in args]
    calc_numbers = sum(number)%26
    print(sum(number)%26)
    return(alphabet[calc_numbers-1])


add_letters('a', 'b', 'c')
add_letters('z')
add_letters('a', 'b')
add_letters('c')
add_letters('z', 'a')
add_letters('y', 'c', 'b')
add_letters()

File: javascript/0006_Alphabetical_Addition.js
function addLetters(...args) {
  const letters = "abcdefghijklmnopqrstuvwxyz";

  const sum = args.reduce((sum, val) => sum + (letters.indexOf(val) + 1), 0);

  const res = (val) => {
    while (val > 26) {
      val = Math.floor(val % 26);
    }
    return val ? val - 1 : 25;
  };

  return letters[res(sum)];
}

console.log(addLetters("a"));
console.log(addLetters("a", "b", "c"));
console.log(addLetters("a", "b"));
console.log(addLetters("z"));
console.log(addLetters("z", "a"));
console.log(addLetters("y", "c", "d"));
console.log(addLetters());
